File: Beginner/Python 3/1008 - Salary.py
# Declare Variables
number = int(input())       # Employer id number
hours_worked = int(input()) # Hours worked per month
hour_money = float(input()) # Money paid per hour

# Calculate salary
salary = hours_worked * hour_money

# Show result
print("NUMBER = {0}\nSALARY = U$ {1:.2f}".format(number, salary))

File: Beginner/C#/1008 - Salary.cs
using System;

class Program
{
    static void Main(string[] args)
    {

        //Declare Variables
        int number = Convert.ToInt32(Console.ReadLine());
        int hoursWorked = Convert.ToInt32(Console.ReadLine());
        double hourlyPay = Convert.ToDouble(Console.ReadLine());
        double salary;

        //Calculate
        salary = hoursWorked * hourlyPay;

        //Print
        Console.WriteLine("NUMBER = {0}", number);
        Console.WriteLine("SALARY = U$ {0:0.00}", salary);

    }
}
